src/program/services/filesystem/vfs/db.py

```
@@ -329,75 +346,87 @@ def get_entry_by_original_filename(
-                    "entry_type": "media",
-                    "url": chosen_url,  # The URL to use for this request
-                }
+                                di[Program].em.add_event(
+                                    Event(
+                                        "RetryLibrary",
+                                        str(item_id),
+                                    )
+                                )
+
+                            raise
+                        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)